yw_website/yw_website/apps/website/templates/includeable/tagarray.html

```diff
@@ -0,0 +1,14 @@
+{% load static %}
+{% get_static_prefix as STATIC_PREFIX %}
+<link rel="stylesheet" type="text/css" href="{{ STATIC_PREFIX }}css/bulma.css">
```

Comment:
this is what i meant by the bulma css link
```
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.4/css/bulma.min.css">
```
it'll work both ways, up to you

Reply:
See above comment